doc/application/index.rst

```diff
@@ -273,7 +275,8 @@ should know about.
   See :ref:`initial-conf` for more information.
 
 * :makevar:`DTC_OVERLAY_FILE`: One or more devicetree overlay files to use.
-  Multiple files can be separated with spaces or semicolons.
+  Multiple files can be separated with spaces using `west` and with semicolons
+  using `cmake`.
```

Comment:
There is no difference between `west` and CMake, so let's just describe one style.
```suggestion
Multiple files can be separated with semicolons.
```